README.md

```diff
@@ -59,10 +63,10 @@ Any workflows that run on a partner image do not fall under the customer [SLA](h
 - Each partner image will have a different term of support depending on the partner, partner support links can be found below
 
 
-| Partner Name | Support Link |
-|-----------|-----------------------|
-| [ARM AVH ](https://resources.github.com/arm-gihub-actions-beta/)| [About & Support](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh.html) |
-
+| Partner  | Image | Support Link | 
+|-----------|-----------|-----------------------|
+| Arm Limited, Inc.| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)| [About & Support](https://github.com/actions/partner-runner-images/issues/new/choose) |
```

Comment:
nit: I recommend adding column `OS / Architecture`:

I will be useful when we add more partner images in future.

| OS / Architecture | Partner  | Image | Support Link | 
|-|-----------|-----------|-----------------------|
| Linux, Arm64 |Arm Limited, Inc.| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)| [About & Support](https://github.com/actions/partner-runner-images/issues/new/choose) |
| Linux, x64 |Arm Limited, Inc.| [ARM AVH](https://resources.github.com/arm-gihub-actions-beta/)| [About & Support](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh.html) |